Extensions/pokes.js

```diff
@@ -136,10 +136,10 @@ XKit.extensions.pokes = {
 
 			if (XKit.extensions.pokes.preferences.allow_fullwidth.value) {
 				poke_html = '<div class="poke ' + poke_class + shiny_class + '" data-pokeid="' + db_nr + '" data-pokesortid="' + poke_sortid + '" data-pokename="' + poke_name + '" data-pokegender="' + poke_gender + '" style="left:' + xpos + 'px; margin-top:' + ypos + 'px;">' +
-				'<img src="' + poke_sprite + '" alt="' + poke_name + '"/>' + '</div>';
+					'<img src="' + poke_sprite + '" alt="' + poke_name + '"/>' + '</div>';
```

Comment:
the `'</div>'` should be on a separate line for consistency, or considering the amount of concatenations here, the entire thing would probably be better off as a template string
(also applies to below string)